packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -3725,6 +3784,7 @@ export default class Meeting extends StatelessWebexPlugin {
         return join;
       })
       .then(async (join) => {
+        // @ts-ignore - config coming from registerPlugin
```

Comment:
Why did you add this?

Reply:
same as before, it is done this way for other this.config. This way, the file is not in red and you can focus in what is really typescript-wrong no?

Reply:
This is trange. `this.config` should work, it is defined in `StatelessWebexPlugin` class but it does not. But `super` works: `super.config.enableAutomaticLLM` (at least no throw error).

I guess the main problem is, the super class is in a JS file.

The other strange thing is, we use named import `import StatelessWebexPlugin from '@webex/webex-core';` but `StatelessWebexPlugin` exported as default

Reply:
Mystery not solved from the sdk :(